.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+	"presets": ["es2015", "stage-2"],
+	"env": {
+		"test": {
+			"plugins": [
+				"istanbul"
+			]
+		}
+	}
+}
```

Comment:
Always add an end of file line. Read why [here](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)